integration-tests/goss/windows/tests/command.goss.yaml

```diff
@@ -7,3 +7,17 @@ command:
     - hello world
     stderr: []
     timeout: 10000
+  wrap a powershell - expect 1:
+    exec: powershell -noprofile -noninteractive -command "(get-itemproperty -path 'HKLM:/SYSTEM/CurrentControlSet/Control/Lsa/').restrictanonymous"
+    exit-status: 0
+    stdout:
+    - 1
+    stderr: []
+    timeout: 10000
+  wrap a powershell - expect 0:
+    exec: powershell -noprofile -noninteractive -command "(get-itemproperty -path 'HKLM:/SYSTEM/CurrentControlSet/Control/Lsa/').restrictanonymous"
+    exit-status: 0
+    stdout:
+    - 0
+    stderr: []
+    timeout: 10000
```

Comment:
There are two tests because I don't know how Travis secures their build machines; _one_ of these should pass, though.

Reply:
makes sense :-)